views/explorer.tmpl

```diff
@@ -16,61 +16,48 @@
-                          class="dropdown text-secondary my-2 {{if lt $blocksCount 20}}disabled{{end}}"
-                          {{if lt $blocksCount 20}}disabled="disabled"{{end}}
+                          class="pagesize text-secondary my-2"
+                      >
+                        <option {{if eq $blocksCount 10}}selected{{end}} value="10">10 per page</option>
+                        <option {{if eq $blocksCount 20}}selected{{end}} value="20">20 per page</option>
+                        <option {{if eq $blocksCount 30}}selected{{end}} value="30">30 per page</option>
+                        <option {{if eq $blocksCount 40}}selected{{end}} value="40">40 per page</option>
+                        <option {{if eq $blocksCount 50}}selected{{end}} value="50">50 per page</option>
+                        {{if eq $blocksCount 10 20 30 40 50}}{{else}}
+                            <option selected value="{{$blocksCount}}">{{$blocksCount}} per page</option>
+                        {{end}}
+                      </select>
+                  </span>
+                  <span class="fs12 nowrap text-right">
+                      <select
+                          data-target="pagenavigation.pageage"
+                          data-action="change->pagenavigation#setPageAge"
+                          data-rows="{{.Rows}}"
+                          data-offsetkey="height"
+                          class="pageage text-secondary my-2"
                       >
-                      {{if ge $pendingBlocks 20}}<option {{if eq $blocksCount 20}}selected{{end}} value="20">20 per page</option>{{end}}
-                      {{if ge $pendingBlocks 30}}<option {{if eq $blocksCount 30}}selected{{end}} value="30">30 per page</option>{{end}}
-                      {{if ge $pendingBlocks 50}}<option {{if eq $blocksCount 50}}selected{{end}} value="50">50 per page</option>{{end}}
-                      {{if ge $pendingBlocks 100}}<option {{if eq $blocksCount 100}}selected{{end}} value="100">100 per page</option>{{end}}
-                      {{if ge $pendingBlocks .WindowSize}}<option {{if eq $blocksCount .WindowSize}}selected{{end}} value="{{.WindowSize}}">{{.WindowSize}} per page</option>{{end}}
-                      {{if eq $blocksCount 20 30 50 100 .WindowSize}}{{else}}<option selected value="{{$blocksCount}}">{{$blocksCount}} per page</option>{{end}}
+                        <option selected disabled>New<option>
+                        {{if lt $topBlock .BestBlock}}
+                            <option value="{{.BestBlock}}">Newest</option>
+                            <option value="{{add $topBlock (int64 $blocksCount)}}">Newer</option>
+                        {{end}}
+                        {{if ge $topBlock .Rows}}
+                            <option value="{{subtract $topBlock (int64 $blocksCount)}}">Older</option>
+                            <option value="{{subtract .Rows 1}}">Oldest</option>
+                        {{end}}
```

Comment:
By making this a dropdown, our no-js users can no longer use this navigation. dcrdata gives high priority to no-js users.

Reply:
@buck54321 it's a dropdown in the designs 
<img width="174" alt="Screen Shot 2019-10-17 at 3 10 54 PM" src="https://user-images.githubusercontent.com/2056512/67007576-5c211500-f0f0-11e9-978d-8c17d48bc3cc.png">


https://xd.adobe.com/view/800a9bfa-dcb9-4985-58fb-ae312bfb80d2-f63b/screen/061c3e84-82df-49ce-8a06-72bc360be393/Explore-blockchain

Reply:
I understand that. The design team largely ignored our discussions on this and other aspects of the layout, which is part of the reason these things need to be discussed before you do them. 

Navigation needs to be accessible to no-js users, which is why I left it as a button instead of a select.

Reply:
Even if this worked without JS, I honestly don't understand the motivation for putting these in a dropdown.  It forces you to click twice.